src/kernel/chainparams.cpp

```diff
@@ -117,8 +117,8 @@ class CMainParams : public CChainParams {
         pchMessageStart[3] = 0xd9;
         nDefaultPort = 8333;
         nPruneAfterHeight = 100000;
-        m_assumed_blockchain_size = 496;
-        m_assumed_chain_state_size = 6;
+        m_assumed_blockchain_size = 540;
+        m_assumed_chain_state_size = 7;
```

Comment:
Getting 502 GB in `blocks/` and 5.1GB in `chainstate/`. The new values seem fine as they should include enough overhead.